daemon/emer-machine-id-provider.c

```diff
@@ -333,35 +325,29 @@ read_machine_id (EmerMachineIdProvider *self)
   g_autoptr(GError) local_error = NULL;
   uuid_t id;
 
-  if (!read_one_machine_id (priv->override_path, id, &local_error))
+  if (!read_one_machine_id (priv->tracking_id_path, id, &local_error))
     {
       if (g_error_matches (local_error, G_IO_ERROR, G_IO_ERROR_NOT_FOUND))
         {
-          g_debug ("Override machine id file %s does not exist, trying default.",
-                   priv->override_path);
+          g_debug ("Override machine id file %s does not exist.",
+                   priv->tracking_id_path);
         }
       else if (g_error_matches (local_error,
                                 EMER_ERROR,
                                 EMER_ERROR_INVALID_MACHINE_ID))
         {
-          g_message ("Failed to read override machine id %s: %s",
-                     priv->override_path,
+          g_message ("Failed to read tracking id %s: %s",
+                     priv->tracking_id_path,
                      local_error->message);
         }
 
-      if (!read_one_machine_id (priv->path, id, &local_error))
-        {
-          g_message ("Failed to read machine id %s: %s",
-                     priv->path,
-                     local_error->message);
-        }
+      return FALSE;
     }
 
   if (uuid_is_null (id))
```

Comment:
This error path is now unneeded because we will only reach this point if `read_one_machine_id` succeeds.

Reply:
addressed